qm.8.md

```diff
@@ -0,0 +1,121 @@
+     Memory: 275.1M (swap max: 0B)
+        CPU: 4.527s
+     CGroup: /QM.slice/qm.service
+             ├─libpod-payload-00de006493bc970788d6c830beb494a58a9a2847a5eda200812d3a8b4e214814
+             │ ├─init.scope
+             │ │ └─993676 /sbin/init
+             │ └─system.slice
+             │   ├─dbus-broker.service
+             │   │ ├─993763 /usr/bin/dbus-broker-launch --scope system --audit
+             │   │ └─993764 dbus-broker --log 4 --controller 9 --machine-id 4ce4c21b211d41e78b7b64418c1c0cb5 -->
+             │   ├─systemd-journald.service
+             │   │ └─993718 /usr/lib/systemd/systemd-journald
+...
+```
+
+## CGROUPS QM.slice
+
+Notice that the QM environment is running systemd and other services within the
+QM.Slice. You can use this slice to modify the cgroups controls of all of the
+processes within the QM environment.
+
+## Install Additional packages in QM
+
+If you want to add other packages into the QM environment, use the `dnf` command
+on the host. For example if you want to install the dnf command you can execute:
+
+```
+# dnf install --installroot=/usr/lib/qm/rootfs dnf
+Updating Subscription Management repositories.
+Unable to read consumer identity
+
+This system is not registered with an entitlement server. You can use subscription-manager to register.
+
+Last metadata expiration check: 0:33:12 ago on Tue 11 Apr 2023 03:42:09 PM EDT.
+Dependencies resolved.
+================================================================================================================
+ Package                                Architecture       Version                     Repository          Size
+================================================================================================================
+Installing:
+ dnf                                    noarch             4.14.0-2.fc38               fedora             478 k
+...
+```
+
+## Entering the QM
+
+You can use the Podman command to enter the QM environment and then actually
+launch containers from with the QM.
+
+```
+sh-5.2# podman exec -ti qm sh
+sh-5.2#
+```
+You can check the SELinux label, by executing the following:
+
+```
+sh-5.2# id -Z
+system_u:system_r:qm_t:s0:c35,c404
+```
+
+Notice that the process is running as `qm_t` indicating that you are in a
+confined QM process within the QM environment.
+
+You can now run containers within the QM using Podman.
+
+```
+sh-5.2# podman run --rm ubi9-minimal echo hi
+Resolved "ubi9-minimal" as an alias (/etc/containers/registries.conf.d/000-shortnames.conf)
+Trying to pull registry.access.redhat.com/ubi9-minimal:latest...
+Getting image source signatures
+Checking if image destination supports signatures
+Copying blob 7bffb309b4e8 done  
+Copying config 96179718b4 done  
+Writing manifest to image destination
+Storing signatures
+hi
+```
+
+It is recommended that you use Podman quadlets to run additional containerized
+apps within the QM. All Apps within the QM are prevented from interfering with
```

Comment:
I'd stick with applications instead of apps throughout, your call.

Reply:
Switched to using applications.